### ISSUE TITLE
Testing of the example notebook fails on Travis

### ISSUE THREAD

**Issue by mberz** [pyfar/pyfar#213]
## General

* pyfar version: develop
* Operating System: Travis

## Description

``` shell
============================= test session starts ==============================

platform linux -- Python 3.7.6, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
cachedir: .tox/examples/.pytest_cache
rootdir: /home/travis/build/pyfar/pyfar, configfile: setup.cfg
plugins: nbmake-0.6
collected 1 item
examples/pyfar_demo.ipynb F                                              [100%]
=================================== FAILURES ===================================
__________________________ examples/pyfar_demo.ipynb ___________________________
Error - No such kernel: 'python3'
------------------------------ Captured log call -------------------------------
WARNING  traitlets:kernelspec.py:291 Kernelspec name python3 cannot be found!
=============================== warnings summary ===============================
.tox/examples/lib/python3.7/site-packages/_pytest/config/__init__.py:1233
  /home/travis/build/pyfar/pyfar/.tox/examples/lib/python3.7/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: collect_ignore

    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
FAILED examples/pyfar_demo.ipynb:: - Error - No such kernel: 'python3'
========================= 1 failed, 1 warning in 0.04s =========================
```

**Comment by alex-treebeard** [comment]
HI @mberz -- apologies, I pushed an update to nbmake removing ipykernel, forgetting that it makes sense bundling an Ipython kernel.

I have since released a fix (nbmake==0.7) This issue should be resolved.

(Feel free to hit me with any feedback and feature reqs)

**Comment by mberz** [comment]
Hi @alex-treebeard, thanks for your comment!
